package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "leaderboard-project",
+  "version": "1.0.0",
+  "description": "- Start with initializing package.json",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] There is no way to start your project at the moment, because the `start` script is not present in your `scripts`. Kindly add `start` script to your `scripts`. Presently, this is what you have under the scripts in your package.json.

![your-script](https://user-images.githubusercontent.com/26597432/201607077-6df13686-4974-447b-8894-ea10b38ea790.png)

- [ ] This is the response I got while trying to start your project.

![response](https://user-images.githubusercontent.com/26597432/201607221-29dbd529-965b-4591-aad3-0cd365c37a99.png)

- [ ] Below is the similitude of what you're expected to have under the `scripts` inside your package.json

![expected](https://user-images.githubusercontent.com/26597432/201607582-9801d554-e3bc-47a8-880c-9b93eeaf9202.png)